Started adding instructions to the shell script files.

gwenwhite opened the pull request:
This is just a rough draft(still trying to master formatting skills in vim as you will see), but hoping to tackle Issue #2.

gwenwhite committed:
Staerted adding instructions to the shell scipt files.

jennyfothergill commented on README.md:
```diff
@@ -106,3 +106,18 @@ Beyond the officially supported [flow environments](https://docs.signac.io/proje
 * Fry
 * Kestrel
+
+### .sh file instructions
```
```suggestion
<details>
     <summary>## Included shell scripts (Click to expand)</summary>
```
I suggest that we keep this heading on the same level as "Cluster support" so that it doesn't seem like a subheading to the previous one. Also I changed the wording to (1) start the heading with a capital letter and (2) give an explanation of what they are vs what extension they have. "Instructions for included shell scripts" could also work but is a little wordy... 🤔 

I think I also like the idea of moving the collapsible part here?

jennyfothergill commented on README.md:
```diff
@@ -106,3 +106,18 @@ Beyond the officially supported [flow environments](https://docs.signac.io/proje
 * Kestrel
+
+### .sh file instructions
+<details>
+     <summary>Clean.sh (Click to expand):</summary>
```
```suggestion
    - clean.sh
```
we could use numbering or bullets here to keep the instructions for the two scripts distinct

jennyfothergill commented on README.md:
```diff
@@ -106,3 +106,18 @@ Beyond the officially supported [flow environments](https://docs.signac.io/proje
+### .sh file instructions
+<details>
+     <summary>Clean.sh (Click to expand):</summary>
+     Clean.sh is a shell script file that will remove the workspace folder, including the files inside of it, within the present planckton-flow clone you are using. To run this shell script:
```
```suggestion
     clean.sh is a shell script file that will remove the workspace folder, including the files inside of it, along with any files matching status.txt, signac*, *.log, *.gsd, or *.out within the present planckton-flow clone you are using. To run this shell script:
```
Added a little more detail

jennyfothergill commented on README.md:
````diff
@@ -106,3 +106,18 @@ Beyond the officially supported [flow environments](https://docs.signac.io/proje
+<details>
+     <summary>Clean.sh (Click to expand):</summary>
+     Clean.sh is a shell script file that will remove the workspace folder, including the files inside of it, within the present planckton-flow clone you are using. To run this shell script:
+    ```bash
+    bash clean.sh 
+     ```
````
```suggestion

    ```bash
    bash clean.sh 
    ```

```
I think these aren't rendering because they need blank lines around them

jennyfothergill commented on README.md:
````diff
@@ -106,3 +106,18 @@ Beyond the officially supported [flow environments](https://docs.signac.io/proje
+    ```bash
+    bash clean.sh 
+     ```
+     in your command line.**MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON_FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM.
````
```suggestion
     in your command line. **THIS WILL DELETE ANY JOBS YOU HAVE RUN SO FAR! MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON-FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM!**
```
fix bold rendering and added more warning

jennyfothergill commented on README.md:
````diff
@@ -106,3 +106,18 @@ Beyond the officially supported [flow environments](https://docs.signac.io/proje
+    bash clean.sh 
+     ```
+     in your command line.**MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON_FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM.
+     <summary>Print_status.sh (Click to expand):</summary>
````
```suggestion
     - print_status.sh
```
if you choose the suggestions above, this is more in line with those

jennyfothergill commented on README.md:
````diff
@@ -106,3 +106,18 @@ Beyond the officially supported [flow environments](https://docs.signac.io/proje
+     ```
+     in your command line.**MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON_FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM.
+     <summary>Print_status.sh (Click to expand):</summary>
+     Print_status.sh is a shell script file that prints out the status of your recently submitted jobs. To run this shell script:
````
```suggestion
     print_status.sh is a shell script file that prints out the status of your recently submitted jobs and saves this information to a file called status.txt. To run this shell script:
```

jennyfothergill commented on README.md:
````diff
@@ -106,3 +106,18 @@ Beyond the officially supported [flow environments](https://docs.signac.io/proje
+     in your command line.**MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON_FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM.
+     <summary>Print_status.sh (Click to expand):</summary>
+     Print_status.sh is a shell script file that prints out the status of your recently submitted jobs. To run this shell script:
+     ```bash
+     bash print_status.sh
+      ```
````
```suggestion

     ```bash
     bash print_status.sh
     ```

```

gwenwhite replied:
Awesome, thanks for the feedback and additions!

jennyfothergill reviewed (commented):
thank you for finding and fixing issues! 🤠

gwenwhite committed:
Update README.md

Fixed the second print_status.sh formatting, but still working on pulling the second drop-down outside of the first drop-down.

gwenwhite committed:
Update README.md

Co-authored-by: Jenny <39961845+jennyfothergill@users.noreply.github.com>

gwenwhite committed:
Update README.md

Co-authored-by: Jenny <39961845+jennyfothergill@users.noreply.github.com>

gwenwhite committed:
Update README.md

Co-authored-by: Jenny <39961845+jennyfothergill@users.noreply.github.com>

gwenwhite committed:
Update README.md

Co-authored-by: Jenny <39961845+jennyfothergill@users.noreply.github.com>

gwenwhite committed:
Update README.md

Co-authored-by: Jenny <39961845+jennyfothergill@users.noreply.github.com>

gwenwhite committed:
Update README.md

Co-authored-by: Jenny <39961845+jennyfothergill@users.noreply.github.com>

gwenwhite committed:
Update README.md

Co-authored-by: Jenny <39961845+jennyfothergill@users.noreply.github.com>

gwenwhite committed:
Update README.md

Co-authored-by: Jenny <39961845+jennyfothergill@users.noreply.github.com>

jennyfothergill commented on README.md:
```diff
@@ -106,3 +106,23 @@ Beyond the officially supported [flow environments](https://docs.signac.io/proje
 * Fry
 * Kestrel
+
+<details>
+     <summary>## Included shell scripts (Click to expand)</summary>
```
```suggestion
## Included shell scripts
```
The formatting here is a little funky, so let's try it back to the way you originally had it!

jennyfothergill commented on README.md:
````diff
@@ -106,3 +106,23 @@ Beyond the officially supported [flow environments](https://docs.signac.io/proje
+
+<details>
+     <summary>## Included shell scripts (Click to expand)</summary>
+    - clean.sh
+     clean.sh is a shell script file that will remove the workspace folder, including the files inside of it, along with any files matching status.txt, signac*, *.log, *.gsd, or *.out within the present planckton-flow clone you are using. To run this shell script:
+
+    ```bash
+    bash clean.sh 
+    ```
+
+     in your command line. **THIS WILL DELETE ANY JOBS YOU HAVE RUN SO FAR! MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON-FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM!**
````
```suggestion
<details>
     <summary>clean.sh (Click to expand)</summary>
clean.sh is a shell script file that will remove the workspace folder, including the files inside of it, along with any files matching status.txt, signac*, *.log, *.gsd, or *.out within the present planckton-flow clone you are using. To run this shell script:

    bash clean.sh 

in your command line. **THIS WILL DELETE ANY JOBS YOU HAVE RUN SO FAR! MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON-FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM!**
</details>
```
trying to fix formatting... things behave strangely with the `<details>` tags

jennyfothergill commented on README.md:
````diff
@@ -106,3 +106,23 @@ Beyond the officially supported [flow environments](https://docs.signac.io/proje
+    ```
+
+     in your command line. **THIS WILL DELETE ANY JOBS YOU HAVE RUN SO FAR! MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON-FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM!**
+  <details>
+     - print_status.sh
+     print_status.sh is a shell script file that prints out the status of your recently submitted jobs and saves this information to a file called status.txt. To run this shell script:
+
+     ```bash
+     bash print_status.sh
+     ```
+
+      in your command line.
````
```suggestion
<details>
     <summary>`print_status.sh` (Click to expand)</summary>
print_status.sh is a shell script file that prints out the status of your recently submitted jobs and saves this information to a file called status.txt. To run this shell script:

     bash print_status.sh

in your command line.
</details>
```

jennyfothergill reviewed (commented):
a couple formatting fixes

jennyfothergill committed:
Update README.md

jennyfothergill committed:
Update README.md

jennyfothergill committed:
Update README.md

jennyfothergill commented on README.md:
```diff
@@ -106,3 +106,21 @@ Beyond the officially supported [flow environments](https://docs.signac.io/proje
+in your command line. **THIS WILL DELETE ANY JOBS YOU HAVE RUN SO FAR! MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON-FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM!**
+</details>
+<details>
+     <summary>`print_status.sh` (Click to expand)</summary>
+print_status.sh is a shell script file that prints out the status of your recently submitted jobs and saves this information to a file called status.txt. To run this shell script:
+
+     bash print_status.sh
+
+in your command line.
```
```suggestion
     <summary>print_status.sh</summary>
`print_status.sh` is a shell script file that prints out the status of your recently submitted jobs and saves this information to a file called status.txt. To run, type the following in your terminal:

     bash print_status.sh

```

jennyfothergill commented on README.md:
```diff
@@ -106,3 +106,21 @@ Beyond the officially supported [flow environments](https://docs.signac.io/proje
 * Kestrel
+
+## Included shell scripts
+<details>
+     <summary>clean.sh (Click to expand)</summary>
+clean.sh is a shell script file that will remove the workspace folder, including the files inside of it, along with any files matching status.txt, signac*, *.log, *.gsd, or *.out within the present planckton-flow clone you are using. To run this shell script:
+
+    bash clean.sh 
+
+in your command line. **THIS WILL DELETE ANY JOBS YOU HAVE RUN SO FAR! MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON-FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM!**
+</details>
```
```suggestion
<details>
     <summary>clean.sh</summary>
`clean.sh` is a shell script file that will remove the workspace folder, including the files inside of it, along with any files matching `status.txt`, `signac*`, `*.log`, `*.gsd`, or `*.out` within the present planckton-flow clone you are using. To run, type the following in your terminal:

    bash clean.sh 

**THIS WILL DELETE ANY JOBS YOU HAVE RUN SO FAR! MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON-FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM!**
</details>
```

jennyfothergill committed:
Update README.md

jennyfothergill committed:
Update README.md

jennyfothergill commented on README.md:
```diff
@@ -106,3 +106,20 @@ Beyond the officially supported [flow environments](https://docs.signac.io/proje
 * Fry
 * Kestrel
+
+## Included shell scripts
+<details>
+     <summary>clean.sh</summary>
+`clean.sh` is a shell script file that will remove the workspace folder, including the files inside of it, along with any files matching `status.txt`, `signac*`, `*.log`, `*.gsd`, or `*.out` within the present planckton-flow clone you are using. To run, type the following in your terminal:
+
+    bash clean.sh 
+
+**THIS WILL DELETE ANY JOBS YOU HAVE RUN SO FAR! MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON-FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM!**
+</details>
+<details>
+     <summary>print_status.sh</summary>
+`print_status.sh` is a shell script file that prints out the status of your recently submitted jobs and saves this information to a file called status.txt. To run, type the following in your terminal:
+
+     bash print_status.sh
+
+</details>
```
```suggestion
## Included shell scripts
<details>
     <summary>clean.sh</summary>
clean.sh is a shell script file that will remove the workspace folder, including the files inside of it, along with any files matching status.txt, signac*, *.log, *.gsd, or *.out within the present planckton-flow clone you are using. To run, type the following in your terminal:

    bash clean.sh 

**THIS WILL DELETE ANY JOBS YOU HAVE RUN SO FAR! MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON-FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM!**
</details>
<details>
     <summary>print_status.sh</summary>
print_status.sh is a shell script file that prints out the status of your recently submitted jobs and saves this information to a file called status.txt. To run, type the following in your terminal:

     bash print_status.sh

</details>
```

jennyfothergill committed:
Update README.md
